support-frontend/assets/pages/supporter-plus-landing/components/oneOffCard.tsx

```diff
@@ -0,0 +1,130 @@
+import { css, ThemeProvider } from '@emotion/react';
+import {
+	from,
+	headline,
+	palette,
+	space,
+	textSans,
+	until,
+} from '@guardian/source/foundations';
+import {
+	Button,
+	buttonThemeReaderRevenueBrand,
+} from '@guardian/source/react-components';
+import { OtherAmount } from '../../../components/otherAmount/otherAmount';
+import { PriceCards } from '../../../components/priceCards/priceCards';
+import { PriceCardsContainer } from '../../../components/priceCards/priceCardsContainer';
+import { PaymentCards } from './PaymentIcons';
+
+const sectionStyle = css`
+	max-width: 600px;
+	margin: auto;
+	background-color: ${palette.neutral[100]};
+	border-radius: ${space[3]}px;
+	padding: 32px 48px ${space[6]}px 48px;
+	${until.desktop} {
+		padding-top: ${space[6]}px;
+	}
+`;
+const titleStyle = css`
+	margin: 0 0 ${space[3]}px;
+	text-align: center;
+	${headline.xsmall({ fontWeight: 'bold' })}
+	${from.tablet} {
+		font-size: 22px;
+	}
+`;
+const standFirst = css`
+	color: #606060;
+	margin-bottom: ${space[2]}px;
+	${from.desktop} {
+		margin-bottom: ${space[3]}px;
+	}
+`;
+
+const btnStyleOverrides = css`
+	width: 100%;
+	justify-content: center;
+	margin-bottom: ${space[6]}px;
+	margin-top: ${space[6]}px;
+`;
+
+const buttonContainer = css`
+	display: flex;
+	flex-direction: column;
+`;
+
+function LandingPageContributionsPriceCards({
```

Comment:
Is this the right name?

Reply:
It's not great!
I've just pushed a commit to refactor this file, as having this as a separate component was unnecessary